README.Rmd

```diff
@@ -13,12 +13,30 @@ knitr::opts_chunk$set(
 )
 ```
 
-# RSimplerEDA
+# RSimplerEDA - A Package to Simplify Some Common Plotting for Exploratory Data Analysis (EDA)
 
 <!-- badges: start -->
 <!-- badges: end -->
 
-The goal of RSimplerEDA is to ...
+
+## Overview
+
+Exploratory Data analysis (EDA) is an important step in any data analysis. 
+However, carrying out EDA with the Altair package requires a lot of coding effort. 
+Moreover, it assumes a basic knowledge of functions and grammar of graphics syntax that are appropriate for visualizing categorical and numerical variables. The simpler_eda package addresses this issue by providing functions that are tailored to produce categorical, numerical and correlation plots using a single line of code. Furthermore, the package provides customization capability for the plots based on specific user needs (theme, title, font, size and etc.). The users are able to spend more time on analyzing the data set and less time configuring Altair plot settings.
```

Comment:
Change all "simpler_eda" to "RSimplerEDA"


less time configuring ggplot settings.